runtime/caml/misc.h

```diff
@@ -503,13 +503,19 @@ extern void caml_instr_atexit (void);
 
 #endif /* CAML_INSTR */
 
-/* Macro used to deactivate thread sanitizer on some functions. */
+/* Macro used to deactivate thread and address sanitizers on some
+   functions. */
 #define CAMLno_tsan
+#define CAMLno_asan
 #if defined(__has_feature)
 #  if __has_feature(thread_sanitizer)
 #    undef CAMLno_tsan
 #    define CAMLno_tsan __attribute__((no_sanitize("thread")))
 #  endif
+#  if __has_feature(address_sanitizer)
+#    undef CAMLno_asan
+#    define CAMLno_asan __attribute__((no_sanitize("address")))
+#  endif
 #endif
```

Comment:
For other readers who might wonder about it:
```
#if defined(__has_feature) && __has_feature(thread_sanitizer)
```
wouldn't work with GCC (and perhaps other non-Clang compilers) because it would try to expand `__has_feature(thread_sanitizer)` even though the left part of `&&` is false.

This justifies the `#define / #undef / #define` dance.